tests/CMakeLists.txt

```diff
@@ -259,9 +263,9 @@ endif()
 #
 message(STATUS "Using C++11")
   if (NOT CMAKE_COMPILER_IS_GNUCXX OR GCC_VERSION VERSION_GREATER 4.6)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++11")
```

Comment:
Is this necessary? We don't use GNU C++ extensions, so `-std=c++11` should be the correct flag.